front50-s3/src/main/java/com/netflix/spinnaker/front50/config/S3Config.java

```diff
@@ -126,4 +126,9 @@ public S3PipelineDAO s3PipelineDAO(ObjectMapper objectMapper, AmazonS3 amazonS3)
   public S3SnapshotDAO s3SnapshotDAO(ObjectMapper objectMapper, AmazonS3 amazonS3) {
     return new S3SnapshotDAO(objectMapper, amazonS3, Schedulers.from(Executors.newFixedThreadPool(5)), 10000, bucket, rootFolder);
   }
+
+  @Bean
+  public S3TaggedEntityDAO s3TaggedEntityDAO(ObjectMapper objectMapper, AmazonS3 amazonS3) {
+    return new S3TaggedEntityDAO(objectMapper, amazonS3, null, -1, bucket, rootFolder);
```

Comment:
Differs slightly from other DAOs in that it will _not_ cache all entities in-memory (_thus no scheduler and -1 refresh interval_)